buildah/Dockerfiles/Makefile

```diff
@@ -0,0 +1,6 @@
+REGISTRY=quay.io/openshift-pipeline/buildah
```

Comment:
We won't have permission to push here right? Maybe we should figure out a home for all the images in the catalog.

Reply:
@vdemeester what do you think ?

Reply:
agreeing with @dlorenc, in the future we may want to have some common place where the image defined here in the catalog are pushed.